src/main/kotlin/org/jitsi/nlj/srtp/SrtpUtil.kt

```diff
@@ -73,6 +73,28 @@ class SrtpUtil {
                         rtpAuthTagLength = 80 / 8
                     )
                 }
+                SRTPProtectionProfile.SRTP_AEAD_AES_128_GCM -> {
+                    SrtpProfileInformation(
+                        cipherKeyLength = 128 / 8,
+                        cipherSaltLength = 96 / 8,
+                        cipherName = SrtpPolicy.AESGCM_ENCRYPTION,
+                        authFunctionName = SrtpPolicy.NULL_AUTHENTICATION,
+                        authKeyLength = 0,
+                        rtcpAuthTagLength = 128 / 8,
+                        rtpAuthTagLength = 128 / 8
+                    )
+                }
+                SRTPProtectionProfile.SRTP_AEAD_AES_256_GCM -> {
```

Comment:
Is this needed? I don't see 256 added to the list above

Reply:
I'm pondering making the cipher list a config option. (Some folks on the community were asking about 256-bit keys, though personally I think it's unnecessary.)